lib/miq_automation_engine/service_models/miq_ae_service_vm.rb

```diff
@@ -1,5 +1,8 @@
 module MiqAeMethodService
   class MiqAeServiceVm < MiqAeServiceVmOrTemplate
+    require_relative "mixins/miq_ae_external_url_mixin"
+    include MiqAeExternalUrlMixin
```

Comment:
@martinpovolny VM was not in the list provided in https://bugzilla.redhat.com/show_bug.cgi?id=1550002#c6.

If a user can set both `remote_console_url=` and `external_url=` what happens?

Reply:
@gmcculloug : in the UI I want to look for the `external_url` first and then as a fallback for the `remote_console_url` (SystemConsole entity). So that backwards compatibility is kept.

Like this:
https://github.com/ManageIQ/manageiq-ui-classic/pull/5688/files#diff-65512d4378cfa992fb57fbceaaf14c59R262

I do not think that we should do things differently for different entities. So moving forward I'd like to deprecate the `remote_console_url`.

Does that make sense to you?

Reply:
Alternatively I'd change the `remote_console_url` method to assign to the same place as `external_url` and add a migration to remove the column. But again, it can be done as a next step.

Reply:
My understand is the `remote_console_url` has a lot more support today for the specific task of setting up a proxy to support VM consoles, but it can also support what `external_url ` is trying to do.  As long as a combined field can support the two use-cases then I'm good.

Reply:
When I added this it was used for this single purpose.

The entity, that it write's to, the `SystemConsole`, that is used for the proxies.

But not this particular column or the particular method `remote_console_url=`

Reply:
Got it, let's create an issue to track the follow items, otherwise I'm good with this change.  👍